tests_integration_with_local_chain/markets/omen/test_omen.py

```
@@ -389,13 +443,13 @@ def test_place_bet_with_autodeposit(
     # Start by moving all funds from wxdai to xdai
     if initial_balances.wxdai > 0:
         WrappedxDaiContract().withdraw(
-            api_keys=test_keys,
+            api_keys=keys,
```

Ruff

Undefined name `keys`

(F821)